doc/api/tls.md

```diff
@@ -1185,11 +1223,27 @@ changes:
     where `ctx` is a `SecureContext` instance. (`tls.createSecureContext(...)`
     can be used to get a proper `SecureContext`.) If `SNICallback` wasn't
     provided the default callback with high-level API will be used (see below).
+  * `pskCallback` {Function}
+     * identity: {string} identity parameter sent from the client.
+     * Returns: {string} pre-shared key that must be a hexadecimal string
+       or null to stop the negotiation process.
+    When negotiating TLS-PSK (pre-shared keys), this function is called
+    with the identity provided by the client. If the server should
+    continue to negotiate PSK ciphers, the return value of this function must be
+    a non-empty string.
```

Comment:
If it returns null, will the server continue to find a lower priority ciphersuite that is acceptable and use that, or will the entire handshake be aborted?

Reply:
I believe it will fail (based on https://www.openssl.org/docs/man1.1.1/man3/SSL_set_psk_use_session_callback.html):
server case - 
> 0 - PSK identity was not found. An "unknown_psk_identity" alert message will be sent and the connection setup fails.

client case -
> Otherwise or on errors callback should return 0. In this case the connection setup fails.

In case of `null` the cpp side will return 0 to the openssl.